lib/bugsnag/integrations/rails/rails_breadcrumbs.rb

```diff
@@ -0,0 +1,134 @@
+require "bugsnag/breadcrumbs/breadcrumbs"
+
+module Bugsnag::RailsBreadcrumbs
+  DEFAULT_RAILS_BREADCRUMBS = [
+    {
+      :id => "perform_action.action_cable",
+      :message => "Perform ActionCable",
+      :type => Bugsnag::Breadcrumbs::PROCESS_BREADCRUMB_TYPE,
+      :allowed_data => [
+        :event_id,
```

Comment:
I don't think you need to include `event_id` (or whatever you might rename it to) in all the breadcrumb types because you add it on after you select the appropriate fields from the notification payload.